presentation-core/src/main/java/tachiyomi/presentation/core/components/SettingsItems.kt

```diff
@@ -115,6 +116,30 @@ fun SortItem(label: String, sortDescending: Boolean?, onClick: () -> Unit) {
     )
 }
 
+@Composable
+fun NondirectionalSortItem(label: String, enabled: Boolean, enabledIcon: ImageVector, onClick: () -> Unit) {
```

Comment:
IconItem has no provision for no icon but would otherwise work for this. I decided this was the least destructive way to do it, but I also don't have a lot of context for that decision so I welcome feedback.